packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -50,6 +50,7 @@ export type AccessRequirementListProps = {
   renderAsModal?: boolean
   numberOfFilesAffected?: number // if provided, will show this instead of the entity information
   requestObjectName?: string // if provided, will show this instead of the entity information or numberOfFilesAffected
+  dialogTitle?: string // if provided, will show this instead of Data Access Request
```

Comment:
I thought this was available at one point, but it's not in there.  Need to override the dialog title in our case!